t/CR373909.csv

```diff
@@ -0,0 +1,146 @@
+purchase_time,currency_code,financial_market_bet_id,bet_type,buy_price,payout_price,sell_price,start_time,underlying_symbol,short_code,is_expired,sell_time
```

Comment:
Right, but this is an _open source_ repo, which is supposed to be a general-purpose _open source_ module for calculating the probability of winning binary options _in general_. It should have _nothing to do_ with Binary.com. There should be nothing Binary.com-specific in this repo.

Remember, when we write open source code, we need to think in an open-source mindset. CPAN is not a place to dump our proprietary code. It is an open-source public world.